pdata/internal/metrics.go

```diff
@@ -258,7 +258,7 @@ func (ms MetricDataPointFlags) SetNoRecordedValue(b bool) {
 	}
 }
 
-// String returns the string representation of the MetricDataPointFlags.
+// Deprecated: [v0.58.0] will be soon removed, no String() on any other pdata.
```

Comment:
Why need to remove this? What the alternative way to get human readable representation of the flags?

Reply:
Why do we need that?

Reply:
Also implementation is completely wrong, since the flags are masks not values. which means you need to mask with that and determine if present, so the human readable is a combination of all properties not one.

Reply:
Sure, but other flags can be added separated by commas once we have more than one flag?

Reply:
I'm ok with deprecating/removing it until there is a real use case